x-pack/plugins/alerting/common/routes/rule/response/schemas/v1.ts

```diff
@@ -336,7 +337,7 @@ export const monitoringSchema = schema.object(
             duration: schema.maybe(
               schema.number({ meta: { description: 'Duration of the rule run.' } })
             ),
-            outcome: schema.maybe(ruleLastRunSchema),
+            outcome: schema.maybe(outcome),
```

Comment:
I guess this is what caused all the json/yaml changes in the files at the top of the PR list.

Can you describe what happened here?  The type was fairly big, and then replaced with just a string?  Did someone have it typed wrong elsewhere?

Reply:
I guess someone used `ruleLastRunSchema` rather than `outcome` by mistake here. (bad side of having multiple schemas...).

`ruleLastRunSchema` is used for `last_run` field in the rule. And `outcome` is also a field of it! 
For the last_run we set it as string [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/alerting/server/task_runner/lib/process_run_result.ts#L85)

And luckily we don't populate that field at all! [see here](https://github.com/elastic/kibana/blob/d2e980d5d45c8e5fc2737c8eb0a4616be7840276/x-pack/plugins/alerting/server/monitoring/rule_monitoring_service.ts#L38)
That is why that schema is not failing...